src/components/FiscalYear/FiscalYearDetails/FiscalYearDetails.js

```diff
@@ -0,0 +1,159 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pane,
+  Row,
+  Col,
+  ExpandAllButton,
+  AccordionSet,
+  Accordion,
+  PaneMenu,
+  Button,
+  MenuSection,
+} from '@folio/stripes/components';
+import { useAccordionToggle } from '@folio/stripes-acq-components';
+
+import {
+  DetailsEditAction,
+} from '../../../common/DetailsActions';
+import ConnectionListing from '../../ConnectionListing';
+import {
+  FISCAL_YEAR_ACCORDION,
+  FISCAL_YEAR_ACCORDION_LABELS,
+} from '../constants';
+import FiscalYearInformation from './FiscalYearInformation';
+
+const FiscalYearDetails = ({
+  fiscalYear,
+  funds,
+  groups,
+  ledgers,
+  onClose,
+  onEdit,
+  openLedger,
+  openGroup,
+  openFund,
+}) => {
+  const [expandAll, sections, toggleSection] = useAccordionToggle();
+
+  const renderActionMenu = useCallback(
+    ({ onToggle }) => (
+      <MenuSection id="fiscal-year-details-actions">
+        <DetailsEditAction
+          perm="finance-storage.fiscal-years.item.put"
+          onEdit={onEdit}
+          toggleActionMenu={onToggle}
+        />
+      </MenuSection>
+    ),
+    [],
```

Comment:
`onEdit` ?

Reply:
`FiscalYearForm` is disconnected due to BE schema changes

Reply:
I still don't think this array should be mepty